docs/references/api_endpoints.txt

```diff
@@ -0,0 +1,3229 @@
+..
+   THIS DOCUMENTATION WAS AUTOMATICALLY GENERATED
+   Do not edit this document directly. Instead, edit the metadata in the api source code.
```

Comment:
"metadata in the api source code" - where is that?

Reply:
It is spread across many files. I'll add an example that will hopefully be explanatory enough. Something like:
```
If you need to edit the documentation for `u.pro.version.v1`, you can find the api source code in `uaclient/api/u/pro/version/v1.py` on the `main` branch of our git repository.
```